app/views/topics/show.html.erb

```diff
@@ -56,12 +55,12 @@
           <% end %>
           <% if post.updated_at > post.created_at %>
             <meta itemprop='datePublished' content='<%= post.created_at.to_formatted_s(:iso8601) %>'>
-            <time itemprop='dateModified' datetime='<%= post.updated_at.to_formatted_s(:iso8601) %>'>
-              <%= post.updated_at %>
+            <time itemprop='dateModified' datetime='<%= post.updated_at.to_formatted_s(:iso8601) %>' class='post-time'>
+              <%= post.updated_at.strftime('%e %b %Y %H:%M:%S %Z') %>
```

Comment:
This page is going to seen by crawlers and used to take the static dump of the HTML page. So if I localize it here, time will be displayed in the format of user/crawler like HTTrack taking the static dump of the website. This can lead to confusion when those static pages are hosted somewhere since timezone won't be present, what do you think @gschlager ?

Reply:
This page is rendered by the server, so it uses the format of the default locale. And it uses the server's time which usually is UTC.

I guess it highly depends on the use case, but I don't think that a lot of users would care about the timezone or even know what UTC is. From the perspective of a user who looks at an archived page it should be good enough. The exact timestamp is contained within the HTML anyway.